### PR TITLE
teamcity: updating to test using Terraform Core 0.15.0

### PR DESCRIPTION
**Pull request by tombuildsstuff**
Depends on #11431

### CODE REVIEW

**Comment by tombuildsstuff** [comment]
Blocked on https://github.com/hashicorp/terraform-plugin-sdk/pull/747

File: .github/workflows/teamcity-test.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v1
         with:
-          java-version: '13'
+          java-version: '8'
```
Comment (tombuildsstuff) [inline .github/workflows/teamcity-test.yaml:21]: need to upgrade/work around this

**Review by katbyte — APPROVED** [review]
LGTM 👍

File: .github/workflows/teamcity-test.yaml (hunk 1 above)
Comment (tombuildsstuff) [reply .github/workflows/teamcity-test.yaml]: 11's a LTS which should be fine

**Pull request merged**